apps/estuary/test/integration/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: '2'
```

Comment:
not sure what this is getting you.
you've already go `divo_kafka` in your dependencies so you can define the service in that single child spec-style line. it also doesn't look like you're actually using this file anywhere.
if you do have a reason to keep the compose file, you're going to want to add a healthcheck to the file to ensure that kafka is fully started and "clustered" appropriately or you'll get unstable behavior interacting with it in your tests.